roles/mongodb/tasks/mongodb_yum.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- include: mongodb_rhel_dependencies.yaml
```

Comment:
Since that file contains only 1 block, maybe we can just move all the contents here?

Just a thought, not a change request. If you feel dedicated file is better,- I'm OK with that.